RocketData.py

```diff
@@ -39,28 +39,27 @@
 #
```

Comment:
Need to cleanup this files comments and todos, since all packets will have timestamps